app/components/generic/table/Table.tsx

```diff
@@ -21,6 +20,7 @@ export default function Table({
   sortOptions = [],
   sortKey = '',
   filters,
+  picker,
```

Comment:
1. `picker` is not a good name here as it can be anything
2. if `onSearch` is enabled with `picker?`, how they will be displayed together on the header? are you expecting to use one at a time?